.travis.yml

```diff
@@ -24,7 +24,9 @@ cache:
   - $HOME/terraform
 env:
   global:
-  - secure: NaiCIm+m5kqdy4Do6s+PnGp/7c/DF6k6N5xbVZ6nhrAbMCBXwVK/jzizw6YzSEHHMnQ+hQUdi95b3lZmv9jYiUXqFmDUJvp66l1Bbn2O1rgVnA5p8gaKyGcxvnVAAQi95LzbEoMjVyEnnu+Q4HAI9cMwh4IMCxKVFvWCbFMwdVqyy1bcp9Tw6jHt2bdwv/SDTvQgmGH+1M233bk/lDV2dHkxgOwmQPwJDEDpjjvJZ1HNe24pgYagiUpDZFrNsj4xiOGjd4rD/fy4f+8OdQFO0RF18O4n4rM0YeIQ24Bj3GOkj4hdUMSYGN2NgMxc1/TxwxyCnCh2xKGb1c/y6Bx7nVCG73jMMpolHBaf1lze83GSnJJaLC9vZ5E07gM72K6ohLDJPAoWZyY3WlgxiM5mXGDcs5poVqhzA7wzPx2WEgCjO+ckXCEnNIc+PksTETcivpkR1tJ0Z6ncY7wNkPVkMY79owMxHbcxyJ8RqWVkNCr21UQOwXOTmu235+IiTJ/nhBo2YNF4oxjcGzPxIkmf6BBfC3i/xY3qwIGsrO09bIgicNMl6mHJ4q2Gfv8jPBPx6dlH4jYhmn5fs5ifrFn9+16a/zIz+nF+EZH3s2ut3QEmwR1LX8SQ39i8ej9LdAT+8OzT6G7o0wTRcZ6tq93Op9mOpXtxCFoY6tFgQZaN0+8=
-  - secure: Dppd5gVPbhjQVPUpYch/gIh+os3D/rDjLgoNyQNuURqsKQK92/LovbSKo7RqU2vz86Q1F4cNt1Bl+IWix3fPBXC1czd8P/fjIcr8rLu53YYFZ50Wh5/LD/D1bgYZurV032gOKL/CAK+iShes1In6qPU3tCcL5Zc0x0my9iMw4ptTg3qjTR576x4uapgZdMYL1UomGNEjTh2KBr1hIL5or/2uckV+kEkc3QHasPJL0h4RFUKtAgC2NDdq90TpQDujrm2zkYPK77BR56Ywvo8NgcaLWbM04OOMG+Eea9baCoERWdNG2dRQGpsi9x+daGCjBZb6+9RKaDw4NqJAa2NaMq3SPwa/uvqX4HhysXtWCHUIIlLfBUJ8lEUo11aoTL2kVBHBKcFvSIqZmRKrVTnu095MqUfEPUPSzip3ZyQ0/gR1Nz3SDwrSWLqpcp2eend1lZTpfFMixidaUhZtmJPFrJDldybI5wX711TGBeLmhX9s1p9EBmgi3G/Tzd7qJrHZQL2TIOe4e505OD/VHZMjeCbwEhN3i8PScZX+9TipJdk8276fYktrxdACeMpb6eAdMndcxTY+aogWdKhhAv8DRf9ltTvPVq7UsSprT1VbSrnZ8xONnxXXvCEk/diyxwA7tSn2rYrZr7bFqXQCLhmY0C/L70I0yCqTpelDtALORYo=
-  - secure: eLN2r6CCGXsnPsBuuwaeDgz3uQzLX2PqgiooKFIckt6YuaEyb0617bOuGMQ0jcxTfAcTYM+mwWibaMUgon2iI0yqCleaA7rWcaf3sR8k0+1BxU09x5tgImgDiDQ33wLBUsblQhuXr8J3a24gyBbM28tJQgrcTaTVMWHBbrbKs9SIy6/YgFl/iT4QeS8OdEOcEz5LFv9cjJE9cB+7SS+CYsUH7cNbIylkQC84jL9QZdulh8BMedAdSiLxNhsw1IT6G+Ve3FoJSBN33E4raK9w388fBILxPLN4bfFR8QteI1Oq+4jrNHF4wME49AnYiSFXG6tIQgrsY6dNA22W1dyLPCjQAWsveJMl4m1o8lmgGORbSjxAWYwtplea3WSoiuQVWIrbchYMKnXEnRzKkGQ8ki7e0vPq56HzMp5IzVV0vQQTB637v7NIolahyzpJID5g+1eOUV72oJaGv9MldEMHRgeIzBH/Akjp1kN/hgnhar6v0HslkrHnXkvxOwOhz/kMtQ/G+DgFg3zXdiegaZLUmEx5DIn6tpN32JReORnB44YE8m6F+EZVwU6eb42uNaURgLB6gooE0q+9NRZ7kn2jMRr8shhc9fFGWG2VC44UludlNOQBN6Vm0U0/1aH5kGxKex4wn1VtTVpSvaZoyWK785zKybwVh1L8LqQiL0AVeWQ=
+  - TRAVIS_AWS_ACCESS_KEY_ID: AKIA2AV2YTL6QKCM4I5F
```

Comment:
Should we remove prefix `TRAVIS_`, the other repositories don't have it

Reply:
OK. We may change the code accordingly.